css-four-column-layout/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container padding-lr">
+    <div class="row">
+      <div class="column-full">
+        <h2 class="text-center">How it works</h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-fourth row justify-align-center wrap">
```

Comment:
No one element should be both a `column-` and a `row`. If you need a `column` to act as a `row`, you should give that element a child with the `row` class.